src/base/dense/owl_base_dense_ndarray_generic.ml

```diff
@@ -3339,10 +3344,8 @@ let conv2d_backward_kernel input kernel stride output' =
               for j = 0 to output_rows - 1 do
                 let in_col = (i * col_stride) + di - pad_left in
                 let in_row = (j * row_stride) + dj - pad_top in
-                if 0 <= in_col
-                   && in_col < input_cols
-                   && 0 <= in_row
-                   && in_row < input_rows
+                if
+                  0 <= in_col && in_col < input_cols && 0 <= in_row && in_row < input_rows
```

Comment:
why does this set of comparisons get pushed onto a single line while the others above are broken out? seems incongruous

Reply:
Is it because the above ones are too long (beyond 80 chars)?